_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -20,15 +20,28 @@ acknowledgements:
 
 The rule applies to any [keyboard event][]: 
  - where the attribute `key` is a [printable character][] key; and
- - the attribute `isComposing` is set to `false`; or the [keyboard event][] is part of an [composition event][] whose attribute `data` includes at least one [non-printable character][] key; and
+ - the method `getModifierState` returns `false`; and
  - is [dispatched][] to an [event target][] within a [HTML document][].
 
-## Expectation
+## Expectation 1
 
 For each test target:
- - an [instrument][] to prevent the outcome of the [keyboard event][] is available; or
- - if the [event target][] does not have [focus], the outcome of the event is prevented; or
- - an [instrument][] is available to force the [keyboard event][] to be part of an [composition event][] whose attribute `data` must include at least one [non-printable character][] key.
+ - an [instrument][] to prevent any result of the [keyboard event][] is available; or
+ - if the [event target][] doesn't have [focus][], any result of the event is prevented; or
+ - an [instrument][] is available to force the [keyboard event][] to have the method `getModifierState` return `true`
+
+**Note:** The same [instrument][] can be used to disable/force more than one event.
+
+**Note:** Preventing results of the event can be done in multiple ways (e.g. removing the event listener; handling the event in a different manner; ...) but the way in which it is done is not relevant for this rule.
+
+## Expectation 2
+
+For each [instrument][] in the set of [instruments][instrument] from Expectation 1, one of the following is true:
+
+- the [instrument][] is in the same [web page][] of the test target; or
+- instructions for the user to locate the [instrument][] are in the same [web page][] of the test target.
```

Comment:
You probably want to use https://github.com/act-rules/act-rules.github.io/blob/54c6814c49cd4ef631444de8a63521f56f6f03e8/pages/glossary/clearly-labeled-location.md